.gitlab/ci/bucket-upload.yml

```diff
@@ -199,6 +205,65 @@ upload-packs-to-marketplace:
     - section_end "Validate Premium Packs"
 
 
+upload-packs-to-marketplace-v2:
+  tags:
+    - gke
+  needs: ["run-validations-upload-flow", "run-unittests-and-lint-upload-flow", "prepare-testing-bucket-mpv2-upload-flow"]
+  stage: upload-to-marketplace
+  artifacts:
+    expire_in: 48 hrs
+    paths:
+      - /builds/xsoar/content/artifacts/*
+    when: always
```

Comment:
Let’s open an issue to remove it once mp-v2 will be ready

Reply:
remove what?

Reply:
We are saving the artifacts in the artifact bucket, so we don't need to keep them for an extended period in Gitlab.